sdk/src/main/java/io/radar/sdk/util/RadarLogBuffer.kt

```diff
@@ -0,0 +1,23 @@
+package io.radar.sdk.util
+
+import io.radar.sdk.Radar
+import io.radar.sdk.model.RadarLog
+
+internal interface RadarLogBuffer {
```

Comment:
There's only one implementation of this interface, so it doesn't seem like an interface is needed.

If you look through the rest of the SDK, you'll see we only use interfaces where it's required.  Helps cut down on boilerplate and keep things really simple.

Reply:
@tjulien this is an artifact of first doing a file-based logging. Having this interface makes the follow-up PR simpler. Ok to remove this later?

Reply:
#151

Reply:
Yup, sounds good 👍